### ISSUE TITLE
Remove ACF as a dependency frontend

### ISSUE THREAD

**Issue by walbo** [DekodeInteraktiv/hogan-core#43]
Would love for themes to be able to disable ACF frontend in order to provide a performance boost.

As far as I can see we only have one line that depends on ACF frontend now. See code below
https://github.com/DekodeInteraktiv/hogan-core/blob/1e566e3ae07413686b36806c8b02acf8de054c4b/includes/class-core.php#L318

**Comment by stian-overasen** [comment]
Agreed, men dette er enklere sagt enn gjort når det kommer til post_meta og flexible content. `get_field()` på et flexible content-felt gjør underverker. Kjør på :)

**Comment by walbo** [comment]
Ja, fant ut at det ikke bare var å benytte `get_post_meta` der. Tenker at vi på sikt bør prøve å få til dette, men ikke noe hast.

**Comment by soderlind** [comment]
For meg som kunde er det hast ;)
